aws-datastore/src/test/java/com/amplifyframework/datastore/storage/sqlite/SQLCommandProcessorTest.java

```diff
@@ -122,7 +124,8 @@ public void rawQueryReturnsResults() throws AmplifyException {
         if (cursor.moveToFirst()) {
             do {
                 Map<String, Object> map = converter.buildMapForModel(cursor);
-                results.add(ModelConverter.fromMap(map, BlogOwner.class));
+                String jsonString = gson.toJson(map);
+                results.add(gson.fromJson(jsonString, BlogOwner.class));
```

Comment:
Don't we have a higher-level utility somewhere that does the object->json->model double conversion, without calling `Gson` directly?

Reply:
Previously, I had added a `ModelConverter::fromMap` method, that did just that.   It was used here, and in the SQLiteStorageAdapter.   I actually removed it in this PR because `fromMap`, which uses Gson, isn't congruent with `toMap`, which uses the `ModelSchema`.    Using Gson does work for this particular use case, but I'm not sure it works for all use cases.